signup/signup.js

```diff
@@ -0,0 +1,69 @@
+var isName, isEmail, isPwd, confirm;
+
+// check full name
+var fullName = document.getElementById("fullname");
+fullName.addEventListener("change", () => {
+    var regName = /^[a-zA-ZÀÁÂÃÈÉÊÌÍÒÓÔÕÙÚĂĐĨŨƠàáâãèéêìíòóôõùúăđĩũơƯĂẠẢẤẦẨẪẬẮẰẲẴẶẸẺẼỀỀỂẾưăạảấầẩẫậắằẳẵặẹẻẽềềểếỄỆỈỊỌỎỐỒỔỖỘỚỜỞỠỢỤỦỨỪễệỉịọỏốồổỗộớờởỡợụủứừỬỮỰỲỴÝỶỸửữựỳỵỷỹ\s\W|_]+$/g;
+    if(!regName.test(fullName.value)) {
+        fullName.nextElementSibling.innerHTML = "Please enter your full name";
+        isName = false;
```

Comment:
```suggestion
        isValidName = false;
```
Nên đặt tên biến có ý nghĩa hơn